### PR TITLE
Publish am-v3.0-psd01

### PR DESCRIPTION
**Pull request by jamsden**

### CODE REVIEW

File: specs/am/architecture-management-shapes.ttl [hunk 1]
```diff
@@ -30,11 +30,11 @@
   rdfs:label "OSLC Architecture Management (AM) Constraints" ;
   dcterms:description "All vocabulary URIs defined in the OSLC Architecture Management (AM) namespace."^^rdf:XMLLiteral ;
   dcterms:publisher <https://open-services.net/about/> ;
-  #dcterms:issued "2020-xx-xx"^^<http://www.w3.org/2001/XMLSchema#date> ;
+  dcterms:issued "2020-09-17"^^<http://www.w3.org/2001/XMLSchema#date> ;
   dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
   dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/am/architecture-management-shapes.ttl> ;
   dcterms:isPartOf <https://oslc-op.github.io/oslc-specs/specs/am/architecture-management-spec.html> ;
```
Comment (berezovskyi) [inline specs/am/architecture-management-shapes.ttl:36]: dcterms:source and dcterms:isPartOf need to be changed to point to their future locations on open-services. See https://github.com/oslc-op/oslc-specs/pull/407/files

**Review by berezovskyi — CHANGES_REQUESTED** [review]
Nice, one more spec to become a PSD. See the comment inside and please add a changelog entry for the PSD, also see https://github.com/oslc-op/oslc-specs/pull/407/files for an example.

**Comment by jamsden** [comment]
We have an agreement with Chet that the change history appendix is no longer needed - we use GitHub for that. This was a hold over from TC specs that didn't use version management. I left the old ones in as part of the transition from the TC to OP, but I don't think its worth maintaining these tables.

File: specs/am/architecture-management-shapes.ttl [hunk 2]
```diff
@@ -30,11 +30,11 @@
   rdfs:label "OSLC Architecture Management (AM) Constraints" ;
   dcterms:description "All vocabulary URIs defined in the OSLC Architecture Management (AM) namespace."^^rdf:XMLLiteral ;
   dcterms:publisher <https://open-services.net/about/> ;
-  #dcterms:issued "2020-xx-xx"^^<http://www.w3.org/2001/XMLSchema#date> ;
+  dcterms:issued "2020-09-17"^^<http://www.w3.org/2001/XMLSchema#date> ;
   dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
   dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/am/architecture-management-shapes.ttl> ;
```
Comment (berezovskyi) [inline specs/am/architecture-management-shapes.ttl:35]: also needs to point to OASIS

File: specs/am/architecture-management-spec.html [hunk 3]
```diff
@@ -872,9 +870,12 @@ <h2>Change History</h2>
         </tr>
         <tr>
           <td>01</td>
-          <td>08/22/2017</td>
```
Comment (berezovskyi) [inline specs/am/architecture-management-spec.html:875]: then let's remove the table altogether to avoid confusion

Comment (berezovskyi) [reply specs/am/architecture-management-spec.html:875]: and the enclosing section, ofc

**Comment by jamsden** [comment]
Unfortunately we have not done that uniformly. Let's ignore this for the PSD and address it in the next revision. If we're going to do this, we need to 1) determine if its essential, and 2) apply it uniformly. I want us to focus on delivery and only do essential changes in these documents going forward.

**Comment by berezovskyi** [comment]
Okay please fix the dcterms source and it’s ready for submission

**Comment by berezovskyi** [comment]
Okay please fix the dcterms source and it’s ready for submission

**Review by berezovskyi — APPROVED** [review]
Please prepare a ZIP and I will file start the process. Just recalled because it's just a a PSD I don't need to wait on Chet.

**Pull request merged**